tutorials/dataframe/df001_introduction.C

```diff
@@ -20,14 +20,8 @@
 void fill_tree(const char *treeName, const char *fileName)
 {
    ROOT::RDataFrame d(10);
-   int i(0);
-   d.Define("b1", [&i]() { return (double)i; })
-      .Define("b2",
-              [&i]() {
-                 auto j = i * i;
-                 ++i;
```

Comment:
This increment seems to be removed.  If it is accurate, it sounds like it would change the behavior the program and hence go beyond the commit log description (which is only "improve syntax") and thus should be explain in the commit log.  If it is not accurate, more details is also needed in the commit to explain why the increment removal is not a change in behavior.

Reply:
I updated the commit message with an explanation for the change, let me know if it seems sufficient.